scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/Readme.md

```diff
@@ -0,0 +1,172 @@
+
+### Package Dimensions
+![dimension example](./documentation/dimension_system.svg)
+- Body size (`body_size_x`, `body_size_y`, `overall_height`) {dimension}
+- Lead dimensions:
+  - Lead width (`lead_width`) {dimension}
+  - Lead length (`lead_len`) {dimension}
+    - Optional for specifying different length for horizontal and vertical leads (`lead_len_H`, `lead_len_V`) {dimension}
+- Lead pitch, currently equal for all sides (`pitch`) {float}
+
+#### Pull Back Leads
+By default the leads are assumed to extend to the body edge. Some components however have the leads pulled back a bit. A typical example are packages that are called LGA by manufacturers.
+These packages need to select the pull back option for the IPC class (`ipc_class: 'qfn_pull_back'`) and need to be able to specify where the lead is.
+
+![dimension example for pull back leads](./documentation/dimension_system_pull_back.svg)
+
+- Most manufacturers give the lead pull back distance (`lead_to_edge`) {dimension}
+   - Alternatively the center position can be given (`lead_center_pos_x` and `lead_center_pos_y`) {dimension}
+   - Another alternative is to give the center to center dimenions for leads. (`lead_center_to_center_x` and `lead_center_to_center_y`) {dimension}
+- Some manufacturers give the body to inner edge dimension instead of the lead length (`body_to_inside_lead_edge`) {dimenions}
+
+### Pad Count
+- Pad count (`num_pins_x`, `num_pins_y`) {int}
+  - `num_pins_x`=0 is used for generating DNF like packages.
+  - `num_pins_y`=0 is used to generate DNF like package footprints but with inverted pin numbering (Mirrored numbering scheme. Some manufactures use this style in their datasheets. Make sure you are not looking at the bottom view before using this. Not supported for QFN and similar.)
+
+### Exposed pad Handling:
+![exposed pad example](../documentation/ep_handling.svg)
+- Size of package exposed pad or slug (`EP_size_x`, `EP_size_y`) {dimension}
+- Size of the footprint pad [optional] (`EP_size_x_overwrite`, `EP_size_y_overwrite`) {float}
+   - Pad size is equal to nominal package pad size if not given.
+   - Use this to create a soldermask-defined pad.
+- Optional the size of the mask cutout (`EP_mask_x`, `EP_mask_y`) {float}
+   - Use to create soldermask-defined pads (in combination with `EP_size_x_overwrite`)
+- Paste is split into a regular grid with (`EP_num_paste_pads`) {[int (x), int (y)]}
+  - The optional paste coverage multiplier determines how much of the exposed copper area is covered by paste. (`EP_paste_coverage`) {float (0..1), default=0.65}
+
+### Rounding of Exposed Pad Corners
+IPC excludes exposed pads from the requirement for rounding its corners. By default the exposed pad does therefore not use rounded corners. Some datasheets do however suggest the use of rounded corners either specified to a specific value or implicitly shown to be equal to the normal pads.
```

Comment:
Perhaps change 'implicitly shown' to 'appearing to be' since the datasheets where you identified this issue are vague and sharing the same radius is really a belief based on the datasheet drawing.

Reply:
Fixed also in gullwing readme